test/test_corpus.cc

```diff
@@ -1046,14 +1050,17 @@ TEST_P(LSPTest, All) {
             for (auto &r : responses) {
                 if (r->isNotification()) {
                     if (r->method() == LSPMethod::SorbetTypecheckRunInfo) {
-                        foundTypecheckRunInfo = true;
                         auto &params = get<unique_ptr<SorbetTypecheckRunInfo>>(r->asNotification().params);
-                        if (assertSlowPath.value_or(false)) {
-                            EXPECT_EQ(params->tookFastPath, false)
-                                << errorPrefix << "Expected Sorbet to take slow path, but it took the fast path.";
-                        }
-                        if (assertFastPath.has_value()) {
-                            (*assertFastPath)->check(*params, test.folder, version, errorPrefix);
+                        // Ignore started messages.
```

Comment:
Nit:  ...and cancelled messages?  Or does the client not get cancelled messages, so that there's only Started and Ended?

Reply:
Ah, good question. Cancellations should not ever happen in `test_corpus` as it uses LSP in single-threaded mode. I'll update the comment.